app/src/main/java/com/github/kulebin/myfoursquareapp/presenter/VenueDetailPresentation.java

```diff
@@ -0,0 +1,11 @@
+package com.github.kulebin.myfoursquareapp.presenter;
+
+import com.github.kulebin.myfoursquareapp.view.VenueDisplayData;
+
+public interface VenueDetailPresentation extends BasePresentation {
+
+    void presentVenueToShowData(VenueDisplayData venueToShowData);
+
+    String getVenueId();
```

Comment:
Leave without any changes for a while,
method getVenueId marked with todo as an action point for the future.